packages/schemas/src/seeds/cloud-api.ts

```diff
@@ -10,8 +10,12 @@ import { adminTenantId } from './tenant.js';
 export const cloudApiIndicator = 'https://cloud.logto.io/api';
 
 export enum CloudScope {
+  // Assigned to the `user` role, which indicates that the user can create a user tenant.
   CreateTenant = 'create:tenant',
+  // Assigned to the `admin:admin` role, which indicates that the admin user can perform arbitrary operations on any tenant.
   ManageTenant = 'manage:tenant',
+  // Assigned to the `user` role, which indicates that the user can update or delete its own tenants.
```

Comment:
No need to mention role here since they are pretty dynamic. Just explain the scope itself would be fine.

Reply:
also use `/* Some comments */` to leverage IDE display